.travis.yml

```diff
@@ -35,6 +35,7 @@ matrix:
           sources:
             - ubuntu-toolchain-r-test
           packages:
+            - g++-8
```

Comment:
@papadop can we avoid this?

Reply:
Strange I do not remember having added this.... At least not on this branch..... I cloned from the master and it tells me that I'm in sync with it... And indeed this was added and then removed on the master.... So I guess we can avoid this.... But I do not understand where this change comes from....